package.json

```diff
@@ -45,6 +45,7 @@
     "@storybook/addon-storysource": "^5.3.13",
     "@storybook/addon-viewport": "^5.3.13",
     "@storybook/react": "^5.3.13",
+    "@testing-library/react": "^9.4.1",
```

Comment:
Why is the update needed here?

Reply:
Previously only the library specific for React hooks was used.

For rendering components or using things like `fireEvent`, we need to use this library.